public/js/actions/LeaderboardActions.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var AppDispatcher = require('../dispatcher/AppDispatcher');
+var AppConstants = require('../constants/AppConstants');
+
+var LeaderBoardActions = {
+
+
+  //Revise to run functions to get User's key, then get leaders, then get pics.  
+
+  getUserKey: function(email, password){
```

Comment:
Correct.  I didn't implement full Flux implementation.  I hooked up the functions in the component.  Refactoring now.